### ISSUE TITLE
Feature request : Search my bookmarks like okular does.

### ISSUE THREAD

**Issue by nazar2sfive** [sumatrapdfreader/sumatrapdf#943]
I want sumatrapdf to give me a search box on top of bookmarks to allow me to search just for bookmarks. This would allow me to quickly search for just the heading/subheading of a chapter.

**Comment by nazar2sfive** [comment]
![image](https://user-images.githubusercontent.com/12841320/62406332-dd750b00-b5c9-11e9-9382-4aac28ae82e4.png)

**Comment by GitHubRulesOK** [comment]
IT should be posible for someone to add the needed alterations to the core code however it seem not to be a trivial request (unless in-part redirecting existing search). I tried to emulate what may be the desired actions using DOS console commands. 

Admitedly this is NOT the way to go, but serves to highlight potential holes in any scope of requests I saved the following code as JUMP.cmd then at dos prompt used jump first few letters of filename then first few letters of bookmark e.g. `Jump Examp Index` (see my test runs below)

     @echo off && FORFILES  /M "%1*.pdf" /C "cmd /c start C:/Progra~1/SumatraPDF/SumatraPDF.exe -named-dest %2 @file"

1) When you search for a bookmark (named destination) the search is often not case sensitive
    (I had issues in the past when there are different bookmarks with different cases.)
    There **needs** to be a similar Case Sensitive/insensitive togle as per document search.

2) When you type partial bookmark entries, the search will stop at the first match,
    There **needs** to be a search Next and search Previous button (same as the main search)

3) My experience suggests that I can use the main search methods almost as quickly as if there was one dedecated just to the bookmark area, however in larger files of more that a few hundred pages, I can appreciate the speed of jumping say 50% before a search starts.

Personally I would suggest such a feature could use the existing function controls but the search **needs** to be flipped between the two canvas areas e.g. it could be tied to using the current F6 toggle to provide a simple "either/or" solution.

![Untitled](https://user-images.githubusercontent.com/12754913/62749403-36262700-ba54-11e9-9fc7-11050646610c.png)

**Comment by nazar2sfive** [comment]
yes please do something about it. searching whole document just for the title takes long time. quick bookmark search would be so much easy.

**Comment by nazar2sfive** [comment]
i have found that wps office first searches bookmarks then start searching form pg1 to end. so bookmark search are first in priority if that is selected.

**Comment by 777vip** [comment]
Eight years have passed, and this problem still hasn't been fixed?

**Comment by RedCodeHero** [comment]
I am also hoping for this feature, coming from okular. Didn't realize how useful it was until I did not have the ability.

**Comment by xiaozhubin** [comment]
I want this feature too.

**Comment by GitHubRulesOK** [comment]
updated finders [(https://github.com/GitHubRulesOK/SumatraPDF-Plus/blob/master/Plus/FindDest.vbs) ](https://github.com/GitHubRulesOK/SumatraPDF-Plus/blob/master/Plus/FindDest.vbs)
```
ExternalViewers [
	[
		CommandLine = "C:\Users\user name\AppData\Local\SumatraPDF\plus\FindDest.vbs" "%1"
		Name = Search Bookmarks
		Filter = *.*
		Key = s
	]
]
```

<img width="778" height="359" alt="Image" src="https://github.com/user-attachments/assets/24133a91-ddac-4093-94c8-e04d2e603b30" />
